src/components/NavigationDrawer/DrawerContent.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import DonutLargeRoundedIcon from '@mui/icons-material/DonutLargeRounded';
+import GroupsIcon from '@mui/icons-material/Groups';
+import ShieldIcon from '@mui/icons-material/Shield';
+import StackedBarChartRoundedIcon from '@mui/icons-material/StackedBarChartRounded';
+import { ButtonBase, SvgIconProps, Toolbar } from '@mui/material';
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+
+const items = [
+  {
+    title: 'Community',
+    Icon: (params: SvgIconProps) => <GroupsIcon {...params} />,
+    href: '/community',
```

Comment:
We can use `paths.ts` file I've mentioned before to get `href` and `title` here instead of hard coding it.